recipes/fica/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "fica" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fica-{{ version }}.tar.gz
+  sha256: aa4e79f827129bdffc6dd2b4dd9f4ca24f4141b6e19311764493874dcdfa0ada
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - pyyaml
+    - docutils
+    - sphinx
+
+test:
+  imports:
+    - fica
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/chrispyles/confit
```

Comment:
Current URL is https://github.com/chrispyles/fica

Reply:
Thanks @chrispyles! Can you also move the required packages to `install_requires` in `setup.py`, instead of reading from `requirements.txt`? That seems to cause tests to fail here.

Reply:
@chrispyles also, can I add your name as maintainer here? :)

Reply:
Can you try again with v0.2.2 and yes happy to be a maintainer